strawberry/experimental/pydantic/object_type.py

```diff
@@ -24,6 +25,9 @@
 
 
 def replace_pydantic_types(type_: Any):
+    origin = getattr(type_, "__origin__", None)
+    if origin is Literal:
+        return type_
```

Comment:
can we add a small comment on why we do this? ^^

Reply:
https://github.com/strawberry-graphql/strawberry/pull/1445/commits/2d43f7649491009ff359c976f500b76ffb12670d - hope this is sufficient!